DISPLAY "(null)" invalid; disabling X11 forwarding
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcuda.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcurand.so locally
INFO:tensorflow:PS hosts are: ['raiders1:2200']
INFO:tensorflow:Worker hosts are: ['raiders1:2201', 'raiders1:2202', 'raiders1:2203', 'raiders1:2204', 'raiders2:2205', 'raiders2:2206', 'raiders2:2207', 'raiders2:2208', 'raiders3:2209', 'raiders3:2210', 'raiders3:2211', 'raiders3:2212', 'raiders8:2213', 'raiders8:2214', 'raiders8:2215', 'raiders8:2216']
E tensorflow/stream_executor/cuda/cuda_driver.cc:491] failed call to cuInit: CUDA_ERROR_NO_DEVICE
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:153] retrieving CUDA diagnostic information for host: raiders3
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:160] hostname: raiders3
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:185] libcuda reported version is: Not found: was unable to find libcuda.so DSO loaded into this program
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:347] driver version file contents: """NVRM version: NVIDIA UNIX x86_64 Kernel Module  367.27  Thu Jun  9 18:53:27 PDT 2016
GCC version:  gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.3) 
"""
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:189] kernel reported version is: 367.27.0
I tensorflow/core/common_runtime/gpu/gpu_init.cc:81] No GPU devices available on machine.
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job ps -> {raiders1:2200}
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job worker -> {raiders1:2201, raiders1:2202, raiders1:2203, raiders1:2204, raiders2:2205, raiders2:2206, raiders2:2207, raiders2:2208, raiders3:2209, raiders3:2210, localhost:2211, raiders3:2212, raiders8:2213, raiders8:2214, raiders8:2215, raiders8:2216}
I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:202] Started server with target: grpc://localhost:2211
INFO:tensorflow:Learning rate: 0.001000, momentum: 0.600000
INFO:tensorflow:Sync mode!!!!!!
INFO:tensorflow:Compute groups : 2
INFO:tensorflow:SyncReplicas enabled: replicas_to_aggregate=16; total_num_replicas=16
INFO:tensorflow:Num nodes per compute groups: 8
INFO:tensorflow:Tokens per step: 8
INFO:tensorflow:2016-07-27 03:15:28.334545 Supervisor
E0727 03:15:44.856721421    7849 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2205': timeout occurred
E0727 03:15:44.856822023    7849 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2206': timeout occurred
E0727 03:15:44.856855410    7849 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2207': timeout occurred
E0727 03:15:44.856870600    7849 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2208': timeout occurred
E0727 03:15:44.981243326    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:15:44.981299025    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:15:44.981313061    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:15:44.981323591    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:15:48.853139762    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:15:48.853185295    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:15:48.853197009    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:15:48.853220219    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:15:54.853081894    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:15:54.853114214    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:15:54.853131812    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:15:54.853143244    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:16:00.853035121    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:16:00.853072908    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:16:00.853085812    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:16:03.989119255    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:16:10.001061405    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:16:13.001021791    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:16:13.001058498    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:16:16.501065784    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:16:26.257046601    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:16:29.257025854    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:16:35.257101754    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:16:35.257146417    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:16:48.009060872    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:16:56.713026358    7552 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
run-expr-32node.sh: line 23:  6897 Terminated              CUDA_VISIBLE_DEVICES='' bazel-bin/inception/imagenet_distributed_train --batch_size=16 --data_dir=/lfs/local/0/daniter/16node-data-$2 --job_name='worker' --task_id=$2 --ps_hosts='raiders1:2200' --worker_hosts='raiders1:2201,raiders1:2202,raiders1:2203,raiders1:2204,raiders2:2205,raiders2:2206,raiders2:2207,raiders2:2208,raiders3:2209,raiders3:2210,raiders3:2211,raiders3:2212,raiders8:2213,raiders8:2214,raiders8:2215,raiders8:2216' --initial_learning_rate=$3 --momentum=$4 --sync=True --compute_groups=$5
